bindings/python/examples/chain.py

```diff
@@ -0,0 +1,169 @@
+    print(f"Swap ID: {swap_id}")
+    print(f"Lockup Address: {response.lockup_details.lockup_address}")
+    print(f"Claim Address: {claim_address}")
+
+    print("\n=== Instructions ===")
+    print(f"1. Send {amount} sats to the lockup address: {response.lockup_details.lockup_address}")
+    print("2. Wait for the swap to be confirmed")
+    print("3. The funds will be sent to the claim address automatically")
+    print("\nMonitoring swap status via WebSocket...")
+
+    asyncio.create_task(ws_client.run_ws_loop())
+
+    lockup_script = boltz_client.SwapScript.from_chain(
+        chain=from_chain,
+        side=boltz_client.Side.LOCKUP,
+        chain_swap_details=response.lockup_details,
+        our_pubkey=refund_keys.public()
+    )
+
+    claim_script = boltz_client.SwapScript.from_chain(
+        chain=to_chain,
+        side=boltz_client.Side.CLAIM,
+        chain_swap_details=response.claim_details,
+        our_pubkey=claim_keys.public()
+    )
+
+    # Monitor the swap status via WebSocket
+    await ws_client.subscribe_swap(swap_id)
+    updates = ws_client.updates()
+    while True:
+        update = await updates.next()
+        status = update.status
+        
+        print(f"\n[{datetime.now().strftime('%H:%M:%S')}] Swap Status: {status}")
+        
+        if status == "swap.created":
+            print("\n=== Action Required ===")
+            print(f"Please send {amount} sats to: {response.lockup_details.lockup_address}")
+            print("Waiting for your transaction...")
+            
+        elif status == "transaction.mempool":
+            print("Transaction detected in mempool!")
+            
+        elif status == "transaction.confirmed":
+            print("Transaction confirmed!")
+            
+        elif status == "transaction.server.mempool":
+            print("Server transaction detected in mempool!")
+            
+        elif status == "transaction.server.confirmed":
+            print("Server transaction confirmed!")
+            
+            # Construct and broadcast claim transaction
+            print("\n=== Constructing Claim Transaction ===")
+
+            claim_params = boltz_client.SwapTransactionParams(
+                output_address=claim_address,
+                fee=boltz_client.Fee.ABSOLUTE(200),
+                swap_id=swap_id,
+                keys=claim_keys,
+                chain_client=chain_client,
+                boltz_client=boltz_api,
+                options=boltz_client.TransactionOptions(
+                    chain_claim=boltz_client.ChainClaim(
+                        keys=refund_keys,
+                        lockup_script=lockup_script
+                    )
+                ),
+            )
+
+            claim_tx = await claim_script.construct_claim(preimage, claim_params)
+            print("Claim transaction constructed, broadcasting...")
+            await chain_client.broadcast_tx(claim_tx)
+            print("Claim transaction broadcasted!")
```

Comment:
```suggestion
            print("Claim transaction broadcast!")
```

Not sure if "broadcasted" is s word